Gemfile

```diff
@@ -35,6 +35,7 @@ gem 'omniauth-facebook'
 gem 'semantic-ui-sass'
 gem 'jquery-rails', '~> 4.3', '>= 4.3.3'
 gem 'rails-ujs', '~> 0.1.0'
+gem 'omniauth-facebook'
```

Comment:
Let make sure to practise DRY code(that is, we shouldn't repeat ourself), we already have this `gem` on line 34, kindly remove one.